pkg/controller/extensions/request/extensionrequest_test.go

```diff
@@ -817,6 +834,7 @@ func TestDiscoverExecutorInfo(t *testing.T) {
 
 			os.Setenv(util.PodNameEnvVar, "podName")
 			os.Setenv(util.PodNamespaceEnvVar, "podNamespace")
+			os.Setenv(PodImageNameEnvVar, tt.imageName)
```

Comment:
Do we still have a test case for when the env var is not set? This seems like it's always setting the env var, but sometimes it happens to be the default `string`.

Reply:
Right, the empty string value is what `Getenv` would return for any variable that does not exist, and that is what the other tests in this section are using.

While it is possible to detect if an environment variable does not exist (`LookupEnv`) that is not used here.  I think a test that verified existence would add no additional coverage for now.  If we start using `LookupEnv` then a test of this nature would cover the paths created by the `bool` returned by that function.